cpp/open3d/data/Dataset.cpp

```diff
@@ -92,44 +89,20 @@ SimpleDataset::SimpleDataset(const std::string& prefix,
     }
 }
 
-namespace dataset {
-
-SampleICPPointClouds::SampleICPPointClouds(const std::string& prefix,
-                                           const std::string& data_root)
+DemoICPPointClouds::DemoICPPointClouds(const std::string& prefix,
+                                       const std::string& data_root)
     : SimpleDataset(
               prefix,
-              {"https://github.com/isl-org/open3d_downloads/releases/"
-               "download/sample-icp-pointclouds/SampleICPPointClouds.zip"},
-              "3ee7a2631caa3c47a333972e3c4fb315") {
+              {"https://github.com/isl-org/open3d_downloads/releases/download/"
+               "20220130-demo-icp-pointclouds/DemoICPPointClouds.zip"},
```

Comment:
Upload all timed dataset into the `20220201-xxx` release tag, such that they are easy to manage in the open3d-downloads dataset.

Reply:
In the release note, add license and original links.